HomeWork/src/main/java/ionio/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package ionio;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public abstract class FileLoggerConfiguration {
+
+    String fileName = "NIO.txt";
+    private LoggingLevel loggingLevel;
+    private LoggingLevel[] arrayLevels;
+    protected int maxSizeFile = 1024;
+    private String formatFile = " [ПОТОЧНИЙ_ЧАС][LEVEL] ";
+
+    protected File logFile = new File(fileName);
+
+    public FileLoggerConfiguration() {
+    }
+
+    public FileLoggerConfiguration(String fileName, LoggingLevel loggingLevel, int maxSizeFile, String formatFile) {
+        if (LoggingLevel.DEBUG.equals(loggingLevel)) {
+            arrayLevels = new LoggingLevel[2];
+            arrayLevels[0] = LoggingLevel.INFO;
+            arrayLevels[1] = LoggingLevel.DEBUG;
+        } else {
+            arrayLevels = new LoggingLevel[1];
+            arrayLevels[0] = LoggingLevel.INFO;
+        }
+        this.fileName = fileName;
+        this.loggingLevel = loggingLevel;
+        this.maxSizeFile = maxSizeFile;
+        this.formatFile = formatFile;
+    }
+
+    public boolean isDebugEnabled() {
+        return contains(LoggingLevel.DEBUG);
+    }
+
+    public boolean isInfoEnabled() {
+        return contains(LoggingLevel.INFO);
+    }
+
+    public boolean contains(LoggingLevel level) {
+        for (int i = 0; i < arrayLevels.length; i++) {
+            if (this.arrayLevels[i].equals(level)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean checkedSizeFile(int sizeContent) throws IOException {
```

Comment:
Remove `throws IOException` everywhere in your PR. Handle this exception yourself.